dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -16,6 +16,28 @@
 
 package io.dockstore.client.cli.nested;
 
+import java.io.File;
```

Comment:
Looking at the codestyle.xml, it looks like my changes are compatible with it.  Perhaps the checkstyle isn't enforcing it.